alkira/resource_alkira_connector_internet_exit.go

```diff
@@ -85,20 +78,49 @@ func resourceConnectorInternetCreate(d *schema.ResourceData, m interface{}) erro
 	}
 
 	d.SetId(id)
-	return resourceConnectorInternetRead(d, m)
+	return resourceConnectorInternetExitRead(d, m)
 }
 
-func resourceConnectorInternetRead(d *schema.ResourceData, m interface{}) error {
+func resourceConnectorInternetExitRead(d *schema.ResourceData, m interface{}) error {
 	return nil
 }
 
-func resourceConnectorInternetUpdate(d *schema.ResourceData, m interface{}) error {
-	return resourceConnectorInternetRead(d, m)
+func resourceConnectorInternetExitUpdate(d *schema.ResourceData, m interface{}) error {
+	client := m.(*alkira.AlkiraClient)
+
+	connector, err := generateConnectorInternetRequest(d, m)
+
+	if err != nil {
+		return err
+	}
+
+	log.Printf("[INFO] Updateing Connector (INTERNET) %s", d.Id())
+	err = client.UpdateConnectorInternetExit(d.Id(), connector)
+
+	return err
 }
 
-func resourceConnectorInternetDelete(d *schema.ResourceData, m interface{}) error {
+func resourceConnectorInternetExitDelete(d *schema.ResourceData, m interface{}) error {
 	client := m.(*alkira.AlkiraClient)
 
 	log.Printf("[INFO] Deleting Connector (internet-exit) %s", d.Id())
```

Comment:
To match other resources
```suggestion
	log.Printf("[INFO] Deleting Connector (INTERNET-EXIT) %s", d.Id())
```